### PR TITLE
routes and links added to correspond with article categories and adde…

### PR DESCRIPTION
**Pull request by jgsn93**
…d home link to header logo

I'd added some of the functionality for this on the previous task already hence the lack of additions and deletions

### CODE REVIEW

**Review by HeatherEver — APPROVED** [review]
Nice work but good to note that you shouldn't be using a .filter() method to filter out the topics as you built an endpoint in your backend to deal with this - so it would be nice to have a request filter the topics rather than a filter.

File: src/components/AllArticlesButtons.jsx [hunk 1]
```diff
@@ -12,25 +12,29 @@ const AllArticlesButtons = ({ setSelectedTopic }) => {
...
+        <LinkContainer to="/articles/coding">
+          <Button variant="info" value="coding">
+            Coding
+          </Button>
+        </LinkContainer>
+        <LinkContainer to="/articles/football">
+          <Button variant="danger" value="football">
+            Football
+          </Button>
+        </LinkContainer>
```
Comment (itskatherine) [inline src/components/AllArticlesButtons.jsx:37]: You could populate this list of category links using a API request that fetches the categories. Imagine if someone added a new category to your website, you'd have to go in and manually change this menu - but if it used a list of categories fetched from the DB you wouldn't have to.
This isn't a manadatory change just something to consider!

File: src/utils/api.js [hunk 2]
```diff
@@ -9,3 +9,9 @@ export const fetchArticles = () => {
     return res.data.articles;
   });
 };
+
+export const fetchArticlesByTopic = (topic) => {
```
Comment (itskatherine) [inline src/utils/api.js:13]: Nice work making the query do the sorting rather than manipulating the data afterwards

**Comment by itskatherine** [comment]
Great stuff Josh, check out my comments and then good to merge 👍 
Sign me up to the Geeson Gazette's mailing list when it's done

**Pull request merged**